### PR TITLE
Starbucks Frontend Clone

### LINKED ISSUE #232: Starbucks Frontend Clone
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

(Write your answer here.)

Starbucks Frontend Clone (UI design using HTML , CSS).

## Motivation

Starbucks clone is a great start for frontend beginners, a lot of motivation towards designing and understanding grids in css.
(Please outline the motivation for the proposal.)

## Pitch

(Please explain why this feature should be implemented and how it would be used.)

This feature will help understanding HTML , CSS easily and effective manner!

### PR DESCRIPTION
**Pull request by dexterousb**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

### Description

(Write your answer here.)

Fixes #232

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: Web Development/Intermediate/Starbucks Frontend Clone/readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Starbucks Homepage Clone
```
Comment (Kushal997-das) [inline Web Development/Intermediate/Starbucks Frontend Clone/readme.md:1]: @dexterousb Add gif about your project in this file .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@dexterousb

File: Web Development/Intermediate/Starbucks Frontend Clone/readme.md (hunk 1 above)
Comment (dexterousb) [reply Web Development/Intermediate/Starbucks Frontend Clone/readme.md:1]: [https://imgur.com/hc1CtTPhttps://imgur.com/hc1CtTP](url)
link to gif file plus added another gif image in project

**Comment by dexterousb** [comment]
I have thereby added the gif of the project , kindly merge the pull request.

**Comment by Kushal997-das** [comment]
@dexterousb Add gif in readme.md file not just link .

**Comment by dexterousb** [comment]
Hi I have added the gif image to the readme file. Kindl merge the pull request! Thanks

**Comment by Kushal997-das** [comment]
@dexterousb Please send a new Pull request !

**Pull request closed without merging**